src/tensor_ops.jl

```diff
@@ -152,17 +157,22 @@ Computes the trace of a second or fourth order tensor.
     @code :(return s)
 end
 
+const vol = trace
```

Comment:
Since `trace` is defined for fourth orders as well this allows to use vol on fourth order tensors. Maybe I should rewrite `vol` as a function: `vol(S::SecondOrderTensor{dim}) = trace(S)` so that vol is only allowed for second order tensors?